cli/base.ts

```diff
@@ -10,6 +10,9 @@ enum EnvNodes  {
 }
 
 export abstract class BaseCommand extends Command {
+
+  static dateDisclaimer = "Date is an approximation and matches to an arbitrary block on the given date"
```

Comment:
```suggestion
  static dateDisclaimer = "Date is an approximation and matches to an arbitrary block near the given date"
```

Reply:
I'll add this, although there's no evidence to suggest this would ever not fall on the date